### PR TITLE
Update migration.md

### PR DESCRIPTION
**Pull request by GabrieleMartini**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fix typo
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by PululuK — CHANGES_REQUESTED** [review]
Hi @GabrieleMartini 
You are on fire 🔥
But, I see that most of your Pr is for the same file. 
Could you please put everything in one PR?

**Comment by GabrieleMartini** [comment]
> Hi @GabrieleMartini
> You are on fire fire
> But, I see that most of your Pr is for the same file.
> Could you please put everything in one PR?

Ok @PululuK thank you

File: src/content/1.7/basics/keeping-up-to-date/migration.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ Make sure to select `dont install demo data` when installing the shop
 
 If you are selling in several countries, additional languages have to be reinstalled on the new shop. This can be done in International > Localization > Import a localization pack.
 
-Also, make sure the IDs of your langs match on the source shop.
+Also, make sure the IDs of your langs matches on the source shop.
```
Comment (matks) [inline src/content/1.7/basics/keeping-up-to-date/migration.md:355]: There are many ID so it's plural => `match` is good, not `matches`

**Review by matks — CHANGES_REQUESTED** [review]

**Comment by matks** [comment]
@GabrieleMartini Thank you for the proof-reading but it seems this PR actually does not fix an issue 😉 so I close it

**Pull request closed without merging**